mqttclients/c/mosquitto_client_extensions/mqtt_callbacks.c

```diff
@@ -40,7 +40,7 @@ void on_connect(
     int rc;
     if ((rc = mosquitto_disconnect_v5(mosq, reason_code, NULL)) != MOSQ_ERR_SUCCESS)
     {
-      printf("Error disconnecting: %s\n", mosquitto_strerror(rc));
+      printf("[ERROR] Failure on disconnect: %s\n", mosquitto_strerror(rc));
```

Comment:
Nit: a macro would be easier to maintain in the future.
E.g. https://github.com/CIPop/azure-sdk-for-c/blob/hfsm/sdk/samples/iot_mosquitto/az_log_listener.h#L30 

Even better, a separate error / info / verbose log system would allow it to be enabled/disabled.

Reply:
Addressed in that PR - I'll have that one go in first so I can use the new logging system in this PR